wazimap_np/demographics.py

```diff
@@ -20,4 +16,14 @@ def get_demographics_profile(geo_code, geo_level, session):
             'values': {'this': total_pop}
         }
     }
+
+    try:
+        religion_data, _ = get_stat_data(['religion_name'], geo_level,
+                                         geo_code, session,
+                                         order_by='-total')
+        most_populous_religion = religion_data[religion_data.keys()[0]]
+        demographic_data['religion_distribution'] = religion_data
+        demographic_data['most_populous_religion'] = most_populous_religion
+    except LocationNotFound:
```

Comment:
As I mentioned in my comment (https://github.com/Code4Nepal/nepalmap_federal/pull/62#issuecomment-513556766), this has the danger of masking inadvertent errors. When our data ingestion fails, this will make it less obvious that we have a problem. Requiring us to include rows for the parks and other areas where there is no official population is more puposeful and won't mask errors in other data.